### PR TITLE
fix(vsc): sideloading error messages

### PR DESCRIPTION
**Pull request by tecton**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/30095282
Effect of clicking "Enable Custom App Upload" button:
use-test-tenant
Effect of error message when local debugging a bot project:
image

### CODE REVIEW

**Review by supkasar — APPROVED** [review]

File: packages/vscode-extension/src/debug/common/debugConstants.ts [hunk 1]
```diff
@@ -130,13 +130,15 @@ export const v3PrerequisiteCheckTaskDisplayMessages: DisplayMessages = {
   summary: "Summary:",
   learnMore: (link: string) =>
     `Visit ${link} to get more info about 'Validate prerequisites' task.`,
-  learnMoreHelpLink: "https://aka.ms/teamsfx-tasks/check-prerequisites",
```
Comment (swatDong) [inline packages/vscode-extension/src/debug/common/debugConstants.ts:133]: Any need to update the aka link target?
Added you to the link owner

**Review by swatDong — DISMISSED** [review]

**Review by HuihuiWu-Microsoft — DISMISSED** [review]
Don't forget to do GDPR classification.

**Comment by MuyangAmigo** [comment]
@tecton Please see this:
* Enable Custom App Upload should redirect to doc.
* Use Test Tenant should open in product page (which talks about how to create test tenant)
image

**Comment by tecton** [comment]
> @tecton Please see this:
> 
> * Enable Custom App Upload should redirect to doc.
> * Use Test Tenant should open in product page (which talks about how to create test tenant)
>   image

@MuyangAmigo  Thanks, I fixed it.

**Review by swatDong — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by supkasar — APPROVED** [review]

**Review by yiqing-zhao — APPROVED** [review]

**Pull request merged**